packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -213,6 +215,45 @@ class MediaQueryData {
   ///   property and how it relates to [padding] and [viewInsets].
   final EdgeInsets viewPadding;
 
+  /// The parts of the display that contain system gestures, typically swipes
```

Comment:
"contains" is kinds strange. Isn't this just the area where the system may detect certain gestures, handle them internally, and not forward those to your app?

Reply:
I will update this to
```
  /// The parts of the display where the system may be detecting particular
  /// gestures.
  ///
  /// This value is typically a drag gesture from an edge of the screen.
```

Let me know if there is any additional improvement/clarification we could make

Reply:
It's important to make this extra clear and part of that will be explaining exactly what AndroidQ does. So:

The areas along the edges of the display where the system consumes certain input gestures and blocks delivery to the app.

Starting with Android Q, simple swipe gestures are used by the system for page navigation and are not delivered to the app. Taps and swipe gestures that begin with a long-press _are_ delivered to the app but simple press-drag-release swipe gestures which begin within the area defined by [systemGestureInsets] are not.

Reply:
There's a little nuance to this when I was testing it yesterday... The swipe gesture can still be registered by the app. I've gotten a drawer to open on a simple swipe, but it simultaneously triggers the back navigation arrow to appear. However, this is not consistent and sometimes, the drawer will appear and almost immediately disappear once the back navigation arrow starts to appear. 

I'll try to share a gif, but I think the general idea is that there will be conflicts in how simple drag gestures are handled in these inset areas and that devs should just avoid it anyway.

Reply:
Did you have a chance to just place a Listener [1] in this area and see what events you actually get in Flutter?

[1] https://master-api.flutter.dev/flutter/widgets/Listener-class.html

Reply:
All this to say, maybe updating the wording to "simple swipe gestures that start within the [systemGestureInsets] areas are used by the system for page navigation and ~are~ may not *be* delivered to the app"

Reply:
Sounds OK to me. We can tighten up the wording in the future, if we need to.

Reply:
@goderbauer I just played with the `Listener`, and it seems like it lets all gestures through and if you start doing a horizontal swipe, it sends a cancel event and begins the system gesture. 

Oddly, if you do a diagonal swipe from the edge (ie. top-left to bottom-right or bottom-left to top-right), that seems to never trigger the system gesture. I decided to try this after reading about it as a "workaround" on some articles on the internet.